prediction_market_agent/agents/think_thoroughly_agent/prompts.py

```diff
@@ -67,6 +67,14 @@
 {sentence}
 Search and scrape the web for information that will help you give a high quality, nuanced answer to the question.
 """
+RESEARCH_OUTCOME_WITH_PREVIOUS_OUTPUTS_PROMPT = """
+Research and report on the following sentence:
+{sentence}
+Search and scrape the web for information that will help you give a high quality, nuanced answer to the question.
+
+You can use the following estimates, but you should not take them as the ground truth, they were generated independently and can be incorrect:
+{previous_scenarios_with_probabilities}
```

Comment:
I couldn't find the function which retrieves `previous_scenarios_with_probabilities` - I imagined this as some kind of memory, is this correct?

Reply:
No specific function, just previous predictions: https://github.com/gnosis/prediction-market-agent/pull/126/files#diff-02d2a6be16ea664e6d4128a79a4d9886a61b3f88c791d3b0bf786a809e720426R214